expenses-manager-frontend/src/main/frontend/src/app/models.ts

```diff
@@ -5,6 +5,7 @@ export interface User {
 }
 
 export interface Expense {
+    id : number;
```

Comment:
asta cred ca ar trebui sa fie optional, pentru ca atunci cand adaugam, nu stim id-ul, este generat in baza de date